### DISCUSSION TITLE
Getting Legal Moves of Opponent

### DISCUSSION THREAD

**Discussion by WafflerIron** [SebLague/Chess-Challenge#473] [Q&A]
Hi, I've been trying to make a bot that is able to think at least a few moves ahead (especially in end-games), however, it doesn't take into account what the opponent's next move will be. Is there a function for generating a the list of legal moves for the opponent?
Thanks

**Comment by Edward-789** [comment]
`getLegalMoves` changes when you use `MakeMove`.
